rotkehlchen/chain/ethereum/modules/convex/convex.py

```diff
@@ -0,0 +1,187 @@
+                    token_amount=lp_token_raw_amount,
+                    token_decimals=18,
+                )
+                pool_tokens_base_amount = Inquirer().find_curve_pool_tokens_amount(
+                    lp_token_address=lp_token_address,
+                )
+                if pool_tokens_base_amount is None:
+                    continue
+                for token, (base_amount, usd_price) in pool_tokens_base_amount.items():
+                    token_amount = base_amount * lp_token_amount
+                    staking_balances[address].append(
+                        AssetBalance(
+                            asset=token,
+                            balance=Balance(
+                                amount=token_amount,
+                                usd_value=token_amount * usd_price,
+                            ),
+                        ),
+                    )
+        return staking_balances
+
+    def on_account_addition(self, address: ChecksumEthAddress) -> Optional[List[AssetBalance]]:
+        """When an account is added for convex check its balances"""
+        staking_balances: List[AssetBalance] = []
+        tokens_to_query = self.staking_pools.keys() & CURVE_POOLS.keys()
+        for lp_token_address in tokens_to_query:
+            pool = self.staking_pools[lp_token_address]
+            try:
+                lp_token_raw_amount = self.ethereum_manager.call_contract(
+                    contract_address=pool.address,
+                    abi=pool.abi,
+                    method_name='balanceOf',
+                    arguments=[address],
+                )
+            except (BlockchainQueryError, RemoteError) as e:
+                log.debug(f'Failed to query convex balance of address {address} for pool {pool.address} due to {str(e)}')  # noqa: E501
+                continue
+            if lp_token_raw_amount == 0:
+                continue
+            lp_token_amount = token_normalized_value_decimals(
+                token_amount=lp_token_raw_amount,
+                token_decimals=18,
+            )
+            pool_tokens_base_amount = Inquirer().find_curve_pool_tokens_amount(
+                lp_token_address=lp_token_address,
+            )
+            if pool_tokens_base_amount is None:
+                continue
+            for token, (base_amount, usd_price) in pool_tokens_base_amount.items():
+                token_amount = base_amount * lp_token_amount
+                staking_balances.append(
+                    AssetBalance(
+                        asset=token,
+                        balance=Balance(
+                            amount=token_amount,
+                            usd_value=token_amount * usd_price,
+                        ),
+                    ),
+                )
+        return staking_balances
+
+    def get_history(
+            self,
+            addresses: List[ChecksumEthAddress],
+            reset_db_data: bool,  # pylint: disable=unused-argument
```

Comment:
why do you pass the argument if is not used? In the `rest.py` you can just avoid passing this argument to the `_api_query_for_eth_module`